### PR TITLE
[0.27] Introduce sensu-service bin stub, update service management scripts to use it

### LINKED ISSUE #174: Refactor sensu bin stubs to act as wrappers for configuring environment
With systemd, smf and other service management systems we can't depend on being able to conditionally set environment variables (e.g. setting PATH and GEM_PATH based on value of EMBEDDED_RUBY).

As a result, we should investigate consolidating any such logic in refactored sensu bin stubs that can set up the environment properly before starting the process, ensure valid configuration for sane restarts, etc. This should give us consistent behavior across platforms without having to maintain the same behavior across multiple service management systems.

Duplicates sensu/omnibus-sensu#8 - code to close this issue will be ported to that repo if completed here first.

Closes #108
Closes #172

### PR DESCRIPTION
**Pull request by cwjohnston**
Closes #174

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -71,7 +71,7 @@ Bunchr::Packages.new do |t|
 
     # all linux platforms are currently using init.d
     # this may change in the future.
-    t.files << '/etc/init.d/sensu-service'
+    t.files << '/etc/init.d/sensu-service-wrapper'
```
Comment (portertech) [inline Rakefile]: Perhaps `-init` would be more appropriate? @cwjohnston @amdprophet WDYT?

File: sensu_configs/init.d/sensu-service-wrapper [hunk 2]
```diff
@@ -12,6 +12,7 @@
 # Description:    Sensu monitoring framework service script
 ### END INIT INFO
 
+short_service=$1
```
Comment (portertech) [inline sensu_configs/init.d/sensu-service-wrapper]: maybe `short_service` should be `sensu_service` and `sensu_service` should be `service_name`?

**Comment by portertech** [comment]
Shall we also update the upstart service files?

File: sensu_configs/init.d/sensu-service-wrapper (hunk 2 above)
Comment (cwjohnston) [inline sensu_configs/init.d/sensu-service-wrapper]: I like the greater specificity of `sensu-service-init`

**Comment by cwjohnston** [comment]
@portertech upstart is done, and i've made changes to the init script as suggested

**Pull request merged**